app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,20 @@
+class BookingsController < ApplicationController
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    if @booking.save
+      redirect_to @booking
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+  def booking_params
+    params.require(:booking).permit(:start_date, :end_date, :car_id, :user_id)
```

Comment:
We don't need User_id in strong params. We'll get the user_id from the login session.